src/components/templates/PartyMap/components/RoomModalOngoingEvent/RoomModalOngoingEvent.tsx

```diff
@@ -54,9 +54,11 @@ export const RoomModalOngoingEvent: React.FC<RoomModalOngoingEventProps> = ({
           </div>
           <div className="artist-ongoing-container">
             <div className="event-title">{eventToDisplay.name}</div>
-            <div>
-              by <span className="artist-name">{eventToDisplay.host}</span>
-            </div>
+            {eventToDisplay.host && (
+              <div>
+                by <span className="artist-name">{eventToDisplay.host}</span>
+              </div>
+            )}
```

Comment:
Why is this change needed here? Is the `host` not always set?

Reply:
reverted